src/main/webapp/scripts/map.js

```diff
@@ -49,31 +69,76 @@ function initMap() {
   map.addListener("click", e => {
     placeMarkerAndPanTo(e.latLng, map);
   });
+  
   /*Add Hard-Coded Markers */
   addMarker(map);
 }
 
-/* Temporary Function to Hold Marker for Sky Garden */
 function addMarker(map) {
-  const skyGarden = {lat: 51.510881, lng: -0.083751}
-  const marker = new google.maps.Marker({
-    position: skyGarden,
+  //hardcoded marker data
```

Comment:
nit, just to be consistent with all the commenting in this file
```suggestion
  /* hardcoded marker data */
```